### PR TITLE
fozzie-components@v3.25.0 - Add launch.json to run / debug individual WDIO tests against chrome / BrowserStack / jest tests #globalconfig

### PR DESCRIPTION
**Pull request by siggerzz**
v3.25.0
------------------------------
*March 23, 2021*

## Added
- launch.json to provide ability to run / debug individual WDIO test specs against chrome / browserstack / jest tests

This gives us the added benefit of being able to use breakpoints within VS Code, rather than relying on WDIO's `browser.debug()` function

Screenshot 2021-03-23 at 15 39 19

Screenshot 2021-03-23 at 16 11 38

### CODE REVIEW

**Review by JoshuaNg2332 — DISMISSED** [review]

**Review by LTurns — DISMISSED** [review]

**Comment by SachaZvetelman** [comment]
Nice, I hope we can find a way to make debugging in VS Code consistent.
Have you tried these commands on a Windows machine to ensure they work there?

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```
Comment (kevinrodrigues) [inline .vscode/launch.json]: Probably doesn't matter too much but this won't be valid json. Might be worth putting this or something similar in the readme instead?

**Review by kevinrodrigues — DISMISSED** [review]

File: .vscode/launch.json (hunk 1 above)
Comment (siggerzz) [reply .vscode/launch.json]: This was autogenerated by VSCode. Happy to remove :)

**Review by DamianMullins — DISMISSED** [review]

File: packages/components/atoms/f-button/jest.config.js [hunk 2]
```diff
@@ -37,8 +37,4 @@ module.exports = {
     ],
 
     testURL: 'http://localhost/',
-
```
Comment (LTurns) [inline packages/components/atoms/f-button/jest.config.js:40]: just need to remove the `cd ../../../../ && yarn allure:clean` from the package.json test script for these components too 👍

Comment (siggerzz) [reply packages/components/atoms/f-button/jest.config.js:40]: done!

**Review by JoshuaNg2332 — DISMISSED** [review]

**Review by LTurns — APPROVED** [review]

**Comment by siggerzz** [comment]
> Nice, I hope we can find a way to make debugging in VS Code consistent.
> Have you tried these commands on a Windows machine to ensure they work there?

Tested and working!

**Review by JoshuaNg2332 — APPROVED** [review]

**Comment by ashleynolan** [comment]
This is awesome – is there any way we can get a `How To` sort of document for using these nice testing features into the storybook documentation?

**Pull request merged**